src/jobs/build_push_image.yml

```diff
@@ -88,6 +81,9 @@ steps:
         - restore_cache:
             name: restore the dependencies cache
             key: <<parameters.cache_name>>
+  - attach_workspace:
+      at: .
```

Comment:
i was not aware that this was missing, meaning this job was quite unuseful if you were building the application within the docker build process. awesome you added it!

may i ask you to document this fact?

Reply:
yea it kinda does not matter
im honestly not 100% sure whats the pros and cons for no-quotes, single-quotes, and double-quotes. though the latter AFAIK converts escape sequences like `\n` whereas single quote does not.

TLDR: not sure which style we prefer. if we agree on something im happy to follow

Reply:
> i am a bit confused why you removed the quotes from the . in the path parameter. at the end the default value is still a string and therefore should be quoted as such.

yea it kinda does not matter
im honestly not 100% sure whats the pros and cons for no-quotes, single-quotes, and double-quotes. though the latter AFAIK converts escape sequences like `\n` whereas single quote does not.

TLDR: not sure which style we prefer. if we agree on something im happy to follow